ospf6d/ospf6_main.c

```diff
@@ -47,6 +47,7 @@
 #include "ospf6_lsa.h"
 #include "ospf6_interface.h"
 #include "ospf6_zebra.h"
+#include "../ospfd/ospf_routemap_nb.h"
```

Comment:
Why do we include from `ospf`, not from `ospf6` here?

Reply:
Changed to #include "ospf6_routemap_nb.h"